C-Plus-Plus/cp/waveArray.cpp

```diff
@@ -0,0 +1,56 @@
+/*	Given an array of integers, return the array which is sorted into a wave like array.
+ *	In other words, arrange the elements into a sequence such that a1 >= a2 <= a3 >= a4 <= a5...
+ *
+ *	Example:
+ *	Given [1, 2, 3, 4]
+ *	One possible answer : [2, 1, 4, 3]
+ *	Another possible answer : [4, 1, 3, 2]
+ */
+
+#include <iostream>
+#include <algorithm>
+#include <vector>
```

Comment:
Remove header with `#include <bits/stdc++.h>`

Reply:
Should I rewrite the header to `#include <bits/stdc++.h>`?